virtcontainers/cgroups.go

```diff
@@ -227,7 +227,7 @@ func (s *Sandbox) constrainHypervisor(cgroup cgroups.Cgroup) error {
 	if err != nil {
 		return fmt.Errorf("failed to get thread ids from hypervisor: %v", err)
 	}
-	if tids == nil || len(tids.vcpus) == 0 {
```

Comment:
You should change the comment below.

Reply:
hmmm, I think `len(tids.vcpus) == 0` is a failure case. Did you add it just for the FC case?

Reply:
fixed.